.github/workflows/build.yml

```
@@ -64,6 +66,7 @@ jobs:
         run: |
           set -e
           source ../.venv/bin/activate
+          source finch-flight-software-env.sh
           source ../zephyr/zephyr-env.sh
```

```suggestion
          source ../.venv/bin/activate
          source ../zephyr/zephyr-env.sh
          source finch-flight-software-env.sh
```

Unless there is a reason why you put `source finch-flight-software-env.sh` between `source ../.venv/bin/activate` and  `source ../zephyr/zephyr-env.sh`.